### ISSUE TITLE
Installation and running the application fails

### ISSUE THREAD

**Issue by prokoudine** [gnome-pomodoro/gnome-pomodoro#565]
I'm trying to build/install the latest git master branch code on Ubuntu 20.04.

When I run `sudo ninja install` from the build directory, I see:

`Installing {my_obfuscated_path}/plugins/sounds/org.gnome.pomodoro.plugins.sounds.gschema.xml to /usr/share/glib-2.0/schemas`

So far so good. But at the very end:

```
Running custom install script '/bin/sh {my_obfuscated_path}/gnome-pomodoro/meson-post-install.sh share'
{my_obfuscated_path}/gnome-pomodoro/meson-post-install.sh: 8: [[: not found
```

And then:

```
$ gnome-pomodoro 
(gnome-pomodoro:214452): GLib-GIO-ERROR **: 21:13:59.031: Settings schema 'org.gnome.pomodoro' is not installed
```

Any pointers much appreciated.

N.B. Ubuntu 20.04 only has GNOME Pomodoro 0.18, it's broken with regards to the lock screen. Someone suggested in your comments section that building the latest from source code fixes that. Hence this attempt.

**Comment by prokoudine** [comment]
Doesn't happen on the autohell build. So I guess it's the meson setup.

**Comment by kamilprusko** [comment]
Thanks for reporting this!

It failed because script was written for bash, and your `/bin/sh` is pointing to other shell. Fixed this compatibility issue in bc7c8a5a417355b4fe781260a828124818a2dbca.
